src/main/java/app/models/person/Person.java

```diff
@@ -0,0 +1,43 @@
+package app.models.person;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@AllArgsConstructor
+@Getter
+public class Person {
+    @Setter
+    UUID id;
+    String name;
```

Comment:
The models are safer to be immutable. Consider making them `private final` and using lombok's `@With` or analog